VideoLocker/res/layout/edx_spinner_dropdown_item.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<CheckedTextView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@android:id/text1"
```

Comment:
Do we need this custom ID, with there being only one View in the layout resource?